src/renderer/util/fileSystem.js

```diff
@@ -43,3 +48,135 @@ export const isSameFileSync = (pathA, pathB, isNormalized=false) => {
   }
   return false
 }
+
+export const moveImageToFolder = async (pathname, image, dir) => {
+  const isPath = typeof image === 'string'
+  if (isPath) {
+    const dirname = path.dirname(pathname)
+    const imagePath = path.resolve(dirname, image)
+    const isImage = isImageFile(imagePath)
+    if (isImage) {
+      const filename = path.basename(imagePath)
+      const extname = path.extname(imagePath)
+      const noHashPath = path.join(dir, filename)
+      if (noHashPath === imagePath) {
+        return imagePath
+      }
+      const hash = getContentHash(imagePath)
+      // To avoid name conflict.
+      const hashFilePath = path.join(dir, `${hash}${extname}`)
+      await fse.copy(imagePath, hashFilePath)
```

Comment:
I think copy should overwrite the existing file.

Reply:
Yes, it will overwrite the existed image, I think it's OK, because it has the same origin path(same hash) and should be the same file?